README.md

```diff
@@ -32,46 +65,73 @@ target :pspdfkit do
 end
 ```
 
-Inside the Appcelerator-iOS project, run `pod install` :
+Once you’ve done that, you can now install the dependencies and builld the plugin:
 
 ```bash
+# Move to the plugin’s directory
 $ cd Appcelerator-iOS/
-$ pod install
-```
 
-And now, build it: 
+# Download the PSPDFKit dependency
+$ pod install
 
-```bash
+# Build the plugin
 $ ti build -p ios --build-only
 ```
 
-If the command above throws an error citing `node-pre-gyp`, [make sure you're using Node 8](https://github.com/nodejs/node-gyp/issues/277): 
-
+Unzip the result into the Titanium folder:
+ 
 ```bash
-$ nvm install 8
-$ nvm use 8
+$ unzip ./dist/com.pspdfkit-iphone-9.2.zip -d ~/Library/Application\ Support/Titanium
```

Comment:
I'd leave the patch version here, because the zip name will use the version specified in the manifest. I would also update the manifest to use the latest stable version of PSPDFKit for iOS.

```suggestion
$ unzip ./dist/com.pspdfkit-iphone-9.2.1.zip -d ~/Library/Application\ Support/Titanium
```